css/style.css

```diff
@@ -0,0 +1,638 @@
+    flex-grow: 0;
+}
+
+.hero__flex-row--two {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0px;
+    gap: 15px;
+    width: 206px;
+    height: 48px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+}
+
+.hero__icn--two {
+    width: 48px;
+    height: 48px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+}
+
+.hero__carbon--ant {
+    position: absolute;
+    width: 48px;
+    height: 47.41px;
+}
+
+.hero__vector {
+    position: absolute;
+    left: 0%;
+    right: 0%;
+    top: 0%;
+    bottom: 0%;
+}
+
+.hero__second--icon {
+    width: 50px;
+}
+
+.clients {
+    width: 1440px;
+    height: 791px;
+    
+    flex: none;
+    order: 2;
+    align-self: stretch;
+    flex-grow: 0;
+    background-image: linear-gradient(
+        to bottom,
+        #FAFAFA 50%,
+        #16697A 40%
+    );
+}
+
+.clients__container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 80px 0px;
+    gap: 50px;
+    position: absolute;
+    width: 1050px;
+    height: 816px;
+    left: calc(50% - 1050px/2);
+}
+
+.clients_row {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0px;
+    width: 864px;
+    height: 164px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+}
+
+.clients__main-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0px;
+    gap: 10px;
+    width: 864px;
+    height: 164px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+}
+
+.clients__h2 {
+    width: 461px;
+    height: 114px;
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 57px;
+    text-align: center;
+    letter-spacing: 0.2px;
+    color: #252B42;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+}
+
+.clientients__paragraph {
+    width: 470px;
+    height: 40px;
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 20px;
+    text-align: center;
+    letter-spacing: 0.2px;
+    color: #737373;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+}
+
+.clients__row-two {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0px;
+    gap: 30px;
+    width: 877px;
+    height: 442px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+}
+
+.clients__card-item {
+    box-sizing: border-box;
+    width: 877px;
+    height: 442px;
+    background: #FFFFFF;
+    border: 1px solid #DEDEDE;
+    border-radius: 5px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    background-image: url('../images/service.jpg');
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.clients__media {
+    position: absolute;
+    left: 0px;
+    right: 0px;
+    top: 0px;
+    bottom: 0px;
+}
+
+.client__filter {
+    position: absolute;
+    height: 382px;
+    left: 0px;
+    right: 0px;
+    top: 409px;
+    background: #16697A;
+}
```

Comment:
Try to remove  all spaces between codes. you can use this vscode extension [here](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)